sdk/eventgrid/eventgrid-namespaces/samples/v1/typescript/package.json

```diff
@@ -32,11 +32,11 @@
   "dependencies": {
     "@azure/eventgrid-namespaces": "latest",
     "dotenv": "latest",
-    "@azure/core-auth": "^1.7.0"
+    "@azure/core-auth": "^1.9.0"
   },
   "devDependencies": {
     "@types/node": "^18.0.0",
-    "typescript": "~5.7.2",
+    "typescript": "~5.6.2",
```

Comment:
why typescript is downgraded?

Reply:
looks like dev-tool uses the version from its own package.json and it has not been upgraded https://github.com/xirzec/azure-sdk-for-js/blob/36f7141af99d2b74980e3a9b988e726453307a59/common/tools/dev-tool/src/util/samples/generation.ts#L228-L229

Reply:
Ah. we should use the same version from the library package, or fix dev-tool issue asap...